zerver/views/custom_profile_fields.py

```diff
@@ -153,6 +153,7 @@ def create_realm_custom_profile_field(
     hint: str = REQ(default=""),
     field_data: ProfileFieldData = REQ(default={}, json_validator=check_profile_field_data),
     field_type: int = REQ(json_validator=check_int),
+    editable_by_user: bool = REQ(default=False, json_validator=check_bool),
```

Comment:
I think the default here should be `True`. The webapp UI checks the input by default on opening the `Add new profile field` form and also I think admins would want the field to be editable by user in most cases.

Reply:
And default of `True` also matches the default in `models.py`.